nautobot_secrets_providers/providers/hashicorp.py

```diff
@@ -15,6 +15,8 @@
 
 __all__ = ("HashiCorpVaultSecretsProvider",)
 
+K8S_TOKEN_DEFAULT_PATH = "/var/run/secrets/kubernetes.io/serviceaccount/token"
```

Comment:
Bandit doesn't like this because it has "secrets" in it. I think you can add `# nosec B105` magic comment to silence that warning for that rule.